etl/src/main/java/org/pih/hivmigration/etl/sql/TreatmentObsMigrator.groovy

```diff
@@ -11,14 +11,14 @@ class TreatmentObsMigrator extends ObsMigrator {
         // the hivmigration_observations table gets created by ObsLoadingMigrator
         // the hivmigration_ordered_other table gets created by StagingTablesMigrator
 
-        migrateProphylaxesState()
+//        migrateProphylaxesState()
         migrateProphylaxesPlan()
-        migrateArtStatus()
-        migrateArtPlan()
-        migrateTbState()
-        migrateTbPlan()
-        migrateCurrentOtherMeds()
-        migrateARTChangeReason()
```

Comment:
@brandones was this intentional?

Reply:
@mogoodrich  Absolutely not. I thought I'd pushed the "fixup" to reset them—I comment them out for testing. My bad. But @cioan please at least skim the code before hitting Approve. (and yeah, it's also my job to read my own PR after submitting—this was a real lapse.)

Reply:
https://github.com/PIH/hivmigration/pull/221